### ISSUE TITLE
Sentinel 控制台权限问题

### ISSUE THREAD

**Issue by Force-King** [alibaba/Sentinel#1190]
提个小建议。。
建议控制台可以分角色提供不同权限。
比如管理员和普通用户。只有管理员可一新增编辑限流降级等规则，普通用户只可以查看 实时监控 和 机器列表 等。否则线上环境使用很危险。

**Comment by sczyh30** [comment]
Good idea. We've introduced ACL interfaces like [AuthService](https://github.com/alibaba/Sentinel/blob/master/sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/auth/AuthService.java), AuthUser and [AuthAction](https://github.com/alibaba/Sentinel/blob/master/sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/auth/AuthAction.java) (introduced in #1042). You may improve the dashboard according to your demand :)

Further discussions can be found in #745. Discussions and contributions are welcomed!

**Comment by Force-King** [comment]
@sczyh30 
这个是我自己改造的控制台，做了简单的权限管理：https://github.com/Force-King/sentinel-dashboard-zk
